app/src/main/java/com/example/pygmyhippo/user/EventFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.example.pygmyhippo.user;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.example.pygmyhippo.R;
+import com.example.pygmyhippo.common.Event;
+
+/**
```

Comment:
Consider adding known issues if any, and what would need to be implemented in the future.